pym/bob/cmds/archive.py

```diff
@@ -486,6 +486,12 @@ def doArchive(argv, bobRoot):
 """.format(subHelp))
     parser.add_argument("-l", "--local", action='store_true',
                         help="ignore the archive configuration and work on the current directory")
+    parser.add_argument("-a", "--all", action='store_true',
+                        help="execute the command on all suitable archives in the user configuration")
+    parser.add_argument("-b", "--backends", type=str, metavar="names",
```

Comment:
There is an `action='append'` mode of argparse. With that, the user could just give `-b` multiple times, one for each archive.